.env

```diff
@@ -1,2 +1,2 @@
-DEV_API=http://localhost:3000
+DEV_API=https://api.stuyspec.com
```

Comment:
Why?

Reply:
ill change it back when it is done. i just don't want to use the local rails.

Reply:
Just don't add this file to git.